script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -493,12 +493,12 @@ That's not the sort of power that anything of human origin should be able to obt
 // not "an extension of humans (an offshoot in a phylogenetic tree)"
 // First clause dropped for flow, if we really think it's necessary will come back and try to fit it in naturally.
 At the end of the day, Dead Apostles are just humans with extremely prolonged lives, right?
- But even a human that's been alive for eons wouldn't be able to pull off a feat like that.
+ But even a human who's been alive for eons wouldn't be able to pull off a feat like that.
 }
 [sha:2eae232a44622ecc70b78e06d824f336d50c2c1a]{
 -- Page 607, Offset 18101.
 -- 祖っていうのは他に何かあるんじゃないのか？」
-Surely there's something else that goes into making an Ancestor?"
+Surely there's something else that goes into the making of an Ancestor?"
```

Comment:
I think the original is more fluid, e.g. you (or at least I) say 
> There's a lot that goes into making a cake

rather than
> There's a lot that goes into the making of a cake